client/src/layouts/AppShellLayout/AppShellLayout.js

```diff
@@ -0,0 +1,38 @@
+import useNotes from "hooks/useNotes";
+import TopBar from "layouts/TopBar";
+import NoteKeeper from "pages/NoteKeeper";
+import { useEffect } from "react";
+
+const AppShellLayout = () => {
+  const {
+    notes,
+    loading,
+    fetchError,
+    operationError,
+    handleCreateNote,
+    handleUpdateNote,
+    handleDeleteNote,
+    handleSearchNotes,
+  } = useNotes();
```

Comment:
We we didn't use useNotes Hook inside NoteKeeper?

Reply:
There is a bug that happens when I do this, and I could not catch why it happens when doing that. I will ask you about it later on and refactor this part based on your answer.

Reply:
Resolved.